src/modules/video_player/index.js

```diff
@@ -19,6 +19,30 @@ function stepPlaybackSpeed(faster) {
     currentPlayer.player.setPlaybackRate(rates[idx]);
 }
 
+const handleVisibilityChange = (function() {
```

Comment:
this closure shouldn't really work as written.. there's no guarantee that the player is loaded when this executes

Reply:
This was rewritten